packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -111,11 +113,13 @@ const getRecords = async (
   type: ObjectType,
   ids?: string[],
   managedBySaltoField?: string,
+  omittedFields: string[] = [],
 ): Promise<RecordById> => {
   const queryableFields = getQueryableFields(type)
+    .filter(field => !omittedFields.includes(apiNameSync(field) ?? ''))
   const typeName = await apiName(type)
   if (_.isEmpty(queryableFields)) {
-    log.debug(`Type ${typeName} had no queryable fields`)
+    log.debug('Type %s had no queryable fields (%d fields omitted)', typeName, omittedFields.length)
```

Comment:
```suggestion
    log.debug('Type %s had no queryable fields or all of its queryable fields were omitted. %o', typeName, {
      queryableFields: queryableFieldNames,
      omittedFields,
    })
```
I think we miss critical info here that will help us debug edge-cases.